engines/sci/detection_tables.h

```diff
@@ -30,6 +30,7 @@ namespace Sci {
 #define GAMEOPTION_SQ4_SILVER_CURSORS       GUIO_GAMEOPTIONS6
 #define GAMEOPTION_EGA_UNDITHER             GUIO_GAMEOPTIONS7
 #define GAMEOPTION_HIGH_RESOLUTION_GRAPHICS GUIO_GAMEOPTIONS8
+#define GAMEOPTION_ENABLE_BLACK_LINED_VIDEO GUIO_GAMEOPTIONS9
```

Comment:
Not all of the games that have been changed have VMD videos, so this option should only be available to the ones that actually havge and use them.

In short, VMD videos are only used in the following games:
- GK2 (full version only)
- LSL7 (full version only)
- Lighthouse (full version only)
- Phantasmagoria 1 (demo and full version)
- RAMA (demo and full version)
- Shivers (full version only)
- SQ6 (full version only)
- Torin's Passage (full version only)

The following DO NOT use VMD videos, so the changes to their detection entries should be reverted:
- GK2 demo: Does not use VMD videos - it uses AVI videos
- KQ7 demo: does not have any video
- KQ7: Does not use VMD videos - it uses AVI and Robot videos
- LSL7 demo: does not have any video
- Lighthouse demo: uses Robot videos
- Mixed Up Mother Goose Deluxe: does not have any video
- Phantasmagoria 2: Does not use VMD videos - it uses AVI DUCK videos
- Police Quest: SWAT (demo and full version): Does not use VMD videos - it uses Robot videos
- Shivers demo: Does not have any video
- SQ6 demo: Does not have any video
- Torin's Passage demo: Does not have any video